.github/workflows/zowe-cli-bundle.yaml

```diff
@@ -129,12 +129,20 @@ jobs:
           ${{ steps.versions.outputs.packages_imperative }} ${{ steps.versions.outputs.packages_cli }}
         mv zowe-node-sdk-typedoc.zip ../zowe-nodejs-sdk-typedoc-${{ env.BUNDLE_VERSION }}.zip
         rm -rf *
-    
+
     - name: Create Python SDK Bundle
       run: |
         mkdir -p temp && cd temp
         mkdir -p licenses && cd licenses && cp /tmp/zowe_licenses_cli.zip zowe_licenses_cli.zip && cd ..
-        pip3 download --no-binary=pyyaml zowe
+        if [[ "${{ inputs.package-tag }}" != "zowe-v1-lts" ]]; then
```

Comment:
I assume that this is basically to avoid adding the pre-release wheels for v1 lts bundles.
😋

Reply:
Correct, I figured for zowe-v1-lts we can keep bundling the old (non-prerelease) wheels from "legacy" branch of Python SDK repo. Not sure it matters since I don't think we link to the zowe-v1-lts artifacts for Python SDK anywhere, but didn't want to risk breaking anything.